runtime/gc_vlhgc/CopyScanCacheListVLHGC.cpp

```diff
@@ -143,7 +143,7 @@ MM_CopyScanCacheListVLHGC::removeAllHeapAllocatedChunks(MM_EnvironmentVLHGC *env
 			MM_CopyScanCacheVLHGC *cache = cacheList->_cacheHead;
 	
 			while(cache != NULL) {
```

Comment:
try to fix some formatting too, in nearby code - missing spacing between 'while' and '('